src/prog/main.F90

```diff
@@ -1047,8 +1051,10 @@ subroutine xtbMain(env, argParser)
    if (runtyp.eq.p_run_path) then
       call prtiming(4,'path finder')
    endif
-   if ((runtyp.eq.p_run_hess).or.(runtyp.eq.p_run_ohess).or.(runtyp.eq.p_run_bhess)) then
+   if (((runtyp.eq.p_run_hess).or.(runtyp.eq.p_run_ohess).or.(runtyp.eq.p_run_bhess)) .and. (mode_extrun.ne.p_ext_turbomole)) then
       call prtiming(5,'numerical hessian')
+   else if (((runtyp.eq.p_run_hess).or.(runtyp.eq.p_run_ohess).or.(runtyp.eq.p_run_bhess)) .and. (mode_extrun.eq.p_ext_turbomole)) then
+      call prtiming(5,'analytical hessian')
```

Comment:
Let's simplify this logic a bit and move the condition for the timer label inside the if block.
```suggestion
   if (((runtyp.eq.p_run_hess).or.(runtyp.eq.p_run_ohess).or.(runtyp.eq.p_run_bhess))) then
      if (mode_extrun.ne.p_ext_turbomole) then
         call prtiming(5,'analytical hessian')
      else
         call prtiming(5,'numerical hessian')
      end if
```
Removing the numerical / analytical attribute would also work.